core/apr doesn't exposes include and lib

`core/apr` is included in the plan.sh as build dependency, but `lib` and `include` path aren't set

```
checking apr_pools.h usability... no
checking apr_pools.h presence... no
checking for apr_pools.h... no
configure: error: cannot find libapr-1 headers
```
Here's some flags for reference:
```
Setting CPPFLAGS=-I/hab/pkgs/core/glibc/2.22/20160612063629/include -I/hab/pkgs/core/glibc/2.22/20160612063629/include -I/hab/pkgs/core/make/4.2.1/20160729193703/include -I/hab/pkgs/core/gcc/5.2.0/20160612064854/include -I/hab/pkgs/core/python2/2.7.12/20160729195942/include -I/hab/pkgs/core/python2/2.7.12/20160729195942/Include -I/hab/pkgs/core/openssl/1.0.2h/20160729194249/include -I/hab/pkgs/core/subversion/1.9.4/20160729221138/include -I/hab/pkgs/core/jdk8/8u92/20160620143238/include -I/hab/pkgs/core/zlib/1.2.8/20160612064520/include
Setting LDFLAGS=-L/hab/pkgs/core/glibc/2.22/20160612063629/lib -L/hab/pkgs/core/gcc-libs/5.2.0/20160612075020/lib -L/hab/pkgs/core/glibc/2.22/20160612063629/lib -L/hab/pkgs/core/gcc-libs/5.2.0/20160612075020/lib -L/hab/pkgs/core/gcc/5.2.0/20160612064854/lib -L/hab/pkgs/core/python2/2.7.12/20160729195942/lib -L/hab/pkgs/core/openssl/1.0.2h/20160729194249/lib -L/hab/pkgs/core/subversion/1.9.4/20160729221138/lib -L/hab/pkgs/core/jdk8/8u92/20160620143238/lib -L/hab/pkgs/core/zlib/1.2.8/20160612064520/lib
```
In the same time `bin` is exported in `PATH`:
```
PATH=/hab/pkgs/altvnk/mesos/1.0.0/20160802142549/bin:/hab/pkgs/core/glibc/2.22/20160612063629/bin:/hab/pkgs/core/coreutils/8.25/20160729192409/bin:/hab/pkgs/core/glibc/2.22/20160612063629/bin:/hab/pkgs/core/coreutils/8.25/20160729192409/bin:/hab/pkgs/core/make/4.2.1/20160729193703/bin:/hab/pkgs/core/gcc/5.2.0/20160612064854/bin:/hab/pkgs/core/patch/2.7.5/20160729193843/bin:/hab/pkgs/core/python2/2.7.12/20160729195942/bin:/hab/pkgs/core/openssl/1.0.2h/20160729194249/bin:/hab/pkgs/core/subversion/1.9.4/20160729221138/bin:/hab/pkgs/core/jdk8/8u92/20160620143238/bin:/hab/pkgs/core/jdk8/8u92/20160620143238/jre/bin:/hab/pkgs/core/apr/1.5.2/20160729211952/bin:......(cut)
```

--- a/apr/plan.sh
+++ b/apr/plan.sh
@@ -1,9 +1,44 @@
 pkg_name=apr
 pkg_origin=core
 pkg_version=1.5.2
-pkg_license=('Apache2')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="Apache Portable Runtime"
+pkg_upstream_url="https://apr.apache.org/"
+pkg_license=('Apache-2.0')
 pkg_source=http://www.us.apache.org/dist/apr/${pkg_name}-${pkg_version}.tar.bz2
 pkg_shasum=7d03ed29c22a7152be45b8e50431063736df9e1daa1ddf93f6a547ba7a28f67a
-pkg_build_deps=(core/gcc core/make core/diffutils core/coreutils core/file)
-pkg_deps=(core/glibc)
+pkg_build_deps=(core/diffutils core/file core/gcc core/iana-etc core/make)
+pkg_deps=(core/gcc-libs core/glibc)
 pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+
+do_prepare() {
+  export LDFLAGS="-lgcc_s ${LDFLAGS}"
+
+  if [[ ! -r /usr/bin/file ]]; then
+    ln -sv "$(pkg_path_for file)/bin/file" /usr/bin/file
+    _clean_file=true
+  fi
+}
+
+do_check() {
+  # If `/etc/services` does not exist, make temporary version from the
+  # `iana-etc` package. This is needed for the testsock tests to pass.
+  if [[ ! -f /etc/services ]]; then
+    cp -v "$(pkg_path_for iana-etc)/etc/services" /etc/services
+    local clean_services=true
+  fi
+
+  make test
+
+  if [[ -n "$clean_services" ]]; then
+    rm -fv /etc/services
+  fi
+}
+
+do_end() {
+  if [[ -n "$_clean_file" ]]; then
+    rm -fv /usr/bin/file
+  fi
+}

